Source/WebCore/page/WindowOrWorkerGlobalScope.cpp

```diff
@@ -39,7 +39,7 @@ void WindowOrWorkerGlobalScope::reportError(JSDOMGlobalObject& globalObject, JSC
 {
     auto& vm = globalObject.vm();
```

Comment:
Here’s an example where we might just want to get rid of this local, since we only need to fetch the VM pointer when the error is not an exception. Not so important, but really an idiom question about how eager we are to always put the VM pointer into a local variable.

Reply:
In this particular function case, L41 RELEASE_ASSERT is using it, so having this here is OK. But I agree generally.